docs/api.md

```diff
@@ -264,6 +265,10 @@ puppeteer.launch().then(async browser => {
 
 Closes browser with all the pages (if any were opened). The browser object itself is considered to be disposed and can not be used anymore.
 
+#### browser.disconnect()
+
+Disconnects Puppeteer from the browser, but leaves the Chromium process running.
```

Comment:
... This is needed to free up resources allocated for browser connection.

(and also a small example of puppeteer.connect + browser.disconnect)